python/ray/tests/test_component_failures_2.py

```diff
@@ -18,8 +18,16 @@ def ray_start_workers_separate_multinode(request):
     num_nodes = request.param[0]
     num_initial_workers = request.param[1]
     # Start the Ray processes.
-    cluster = Cluster()
-    for _ in range(num_nodes):
+    cluster = Cluster(
+        initialize_head=True,
+        head_node_args={
+            "num_cpus": num_initial_workers,
+            "_system_config": {
+                # disable worker capping
+                "kill_idle_workers_interval_ms": -1
```

Comment:
It's not for debugging. The `test_worker_failed` case intentionally kills all workers. If worker capping is on, "OSError: [Errno 3] No such process" may be thrown. I've updated the code by ignoring `OSError` instead of turning off worker capping.